mkdocs/docs/HPC/getting_started.md

```diff
@@ -0,0 +1,209 @@
+
+{%- if OS == windows %}
+
+The [HPC-Ugent web portal](https://login.hpc.ugent.be) provides a filebrowser that allows uploading files.
+For more information see the [file browser section](web_portal.md#file-browser).
+
+Upload the two files to your **Home Directory** and go back to your shell.
+
+!!! Info
+
+    As an alternative, you can use WinSCP (see [our section](connecting.md#winscp))
+
+{%- else %}
+
+On your local machine you can run:
+<pre><code>wget https://raw.githubusercontent.com/EESSI/eessi-demo/main/TensorFlow/TensorFlow-2.x_mnist-test.py
+wget https://raw.githubusercontent.com/EESSI/eessi-demo/main/TensorFlow/run.sh
+</code></pre>
+
+Now using `scp` the files can be copied from your local host to your *home directory* (~) on the remote host (HPC).
+<pre><code>scp TensorFlow-2.x_mnist-test.py run.sh {{userid}}{{ loginnode }}:~ </code></pre>
+<pre><code>ssh  {{userid}}@{{ loginnode }} </code></pre>
+
+!!! Info
+
+    For more information about transfering files or `scp`, see [tranfer files from/to hpc](connecting.md#transfer-files-tofrom-the-hpc).
+
+{%- endif %}
+
+When running `ls` on your HPC shell, you should see the two files listed:
+
+```shell
+$ ls
+run.sh TensorFlow-2.x_mnist-test.py
+```
+
+!!! Warning
+
+    When you do not see these files, make sure you uploaded the files to your **Home Directory**.
+
+### Submitting a job
+
+Jobs are submitted and executed using job scripts. In our case **run.sh**.
+
+A job script is a shell script, a text file that specifies the resources, software that is used (via module load) and the steps that should be done for the calculation.
+
+Our job script looks like this:
+
+<center>-- run.sh --</center>
+
+```bash
+#!/bin/bash
+
+module load TensorFlow/2.3.1-foss-2020a-Python-3.8.2
+
+time python TensorFlow-2.x_mnist-test.py
+
+```
+<sub>As you can see this job script will run the python program: **TensorFlow-2.x_mnist-test.py**</sub>
+
+
+The jobs you submit are per default executed on **cluser/{{defaultcluster}}**, you can swap cluster by the following command.
+
+<pre><code>module swap cluster/{{othercluster}}</code></pre>
+
+
+This job script can now be submitted to the cluster's job system for execution, using the qsub (**q**ueue **sub**mit) command:
+
+```shell
+$ qsub run.sh
+{{jobid}}
+```
+
+This command returns a job identifier (*{{jobid}}*) on the HPC cluster. this is a unique identifier for the job and can be used to monitor and manage your job.
+
+!!! Warning
+ 
+    Please take note that the module commands exclusively modify environment variables. For instance, using `module swap cluster/{{othercluster}}` will instruct `qsub` to carry out the task within the {{othercluster}} cluster. 
+    It's important to understand that while these commands affect jobs, they do not alter the context of commands executed directly in your shell. These commands will still operate on your login cluster: {{defaultcluster}}.
```

Comment:
This is not really correct, there's no "login cluster", only "login nodes" and a "default cluster".

I would change it to:

```suggestion
It is important to understand that while `module` commands affect your session environment, they do not change *where* the commands you are running are being executed: they will still be run on the login node you are on. When submitting a job script, the commands *in* the job script will be run on a workernode of the cluster the job was submitted to (like `{{defaultcluster}}`).